src/gazebo_geotagged_images_plugin.cpp

```diff
@@ -201,10 +202,12 @@ void GeotaggedImagesPlugin::OnUpdate(const common::UpdateInfo& /*_info*/) {
   last_time_ = current_time;
 }
 
-void GeotaggedImagesPlugin::OnNewGpsPosition(ConstVector3dPtr& v)
-{
-  lastGpsPosition_ = *v;
-  //gzdbg << "got gps pos: "<<lastGpsPosition_.x()<<", "<<lastGpsPosition.y()<<endl;
+void GeotaggedImagesPlugin::OnNewGpsPosition(GpsPtr& gps_msg) {
+  lastGpsPosition_.x = gps_msg->latitude_deg();
+  lastGpsPosition_.y = gps_msg->longitude_deg();
+  lastGpsPosition_.z = gps_msg->altitude();
+  gzerr << "NEW GPS POS";
```

Comment:
@bkueng This gets never called - do you see why this could be?

Reply:
I got it to work when I initialize the node handle without providing the namespace: `node_handle_->Init();` I'm not sure what the 'proper' way is though.

Reply:
I reworked the commit & removed the debug printf's